doc/_static/custom_styles.css

```diff
@@ -160,3 +160,44 @@ div#driver-popover-item .tour-progress-indicator {
 a[href="#join-our-community"] + ul li.toc-h3 {
   display: none;
 }
+
+/*Version selector and header*/
+/* #versions-header {
+  margin: 0 10px;
+}
+
+#select-versions, #select-versions option {
+  width: 90%;
+}
+
+#select-versions {
+  margin: 10px;
+} */
+
+#versions-header {
+  color: rgba(0,0,0,.65);
+  font-size: 0.8rem;
+  text-transform: uppercase;
+  margin-top: 1.1rem;
+}
+
+#select-versions {
+  margin-bottom: 7px;
+  border: 1px solid rgba(0, 0, 0, 0.15);
+  font-size: 90%;
+  color: rgba(0,0,0,.65);
+  padding: 2px;
+}
+
+#select-versions, #select-versions option {
+  width: 98%;
+}
+
+.form-control {
+  box-shadow: none;
+  background: transparent;
+  border: 1px solid rgba(0, 0, 0, 0.15);
+  height: 54px;
+  font-size: 18px;
+  font-weight: 300;
```

Comment:
@dalthviz, this change didn't take effect in Netlify. So, if every one agrees, I'm going to remove it and add it to the theme repo.

I was not expecting for my suggestions to be pushed by @dalthviz without discussion, but since it seems you liked them, I have no problem in doing that.

Reply:
I did a change (load the custom css at the end) and now the changes are being applied, however it looks kind of strange. Could you check @ccordoba12 and maybe provide the correct style rules/missing style rules if needed? This is what I'm seeing using the Netlify preview:

![image](https://user-images.githubusercontent.com/16781833/91369810-4f6e5280-e7d2-11ea-915e-ab36d6ac6b04.png)

If that is the correct proposal then I think it could look better if we drop the `font-size` and `height` rules and maybe add `text-transform` to uppercase. A preview:

With `text-transform`

![image](https://user-images.githubusercontent.com/16781833/91369772-39609200-e7d2-11ea-9d7b-80a7b649e8a8.png)

Without `text-transform`

![image](https://user-images.githubusercontent.com/16781833/91369915-95c3b180-e7d2-11ea-859e-c8455f353044.png)

Reply:
> I did a change (load the custom css at the end) and now the changes are being applied, however it looks kind of strange

@dalthviz, I think we shouldn't mess with that at this point, i.e. when we are about to do the final release. Not only our selected font is missing, but the style for paragraphs and links is different now:

![Selección_065](https://user-images.githubusercontent.com/365293/91371425-90686600-e7d6-11ea-9bb5-97fe631194c6.png)

I think we can explore this possibility after the release, but for now let's keep working as we've done it so far. Do you agree? And if so, could you revert the change you made?

Reply:
Realized I never submitted this comment, sorry.

The problem is that `custom_styles.css` from the Docs repo (and the others listed in the custom styles key, as well as Pygments and Sphinx-Dropdown, which I'm _guessing_ is from Sphinx-Panels), gets injected _before_ the styles from the theme, `theme.css` and `spyder.css`, such that it doesn't override the theme's stylesheets as it should. They are injected [here](https://github.com/spyder-ide/spyder-docs-sphinx-theme/blob/6bec8d2434e783212c3347395fb6f8d8a19b2e30/pandas_sphinx_theme/layout.html#L18), while the theme.css and spyder.css are included directly below.

We could simply move the inject below the theme styles, [EDIT: That's what you did] but that wouldn't work on its own, because the `sphinx-bootstrap.css` stylesheet from `theme.conf` is also loaded in this stack and that would move it above the theme stylesheets where it shouldn't be; while our page doesn't actually use any of those styles (non-overriden at least, I've investigated every last one using Chrome's coverage and careful manual checks of both the page's appearance and the styles applied to every element it could potentially affect), it does use plenty from `basic.css` that this stylesheet `@import`s, and those need to be below the theme styles.

What I suggest instead that would preserve the current order in which styles are applied (except for the custom_styles CSS in question, which was always intended to go after the theme styles), as well as unify the theme variables (color, etc) as discussed last night, is `@import` or `@use` basic.css in `theme.css`, `@use` the `theme.scss` stylesheet in `spyder.scss`, remove the completely unused `sphinx-bootstrap.css` stylesheet and replace it with `spyder.css` in `theme.conf`, and remove the now-unneeded manual loading of `theme.css` and `spyder.css` in `layout.html`. Besides fixing the loading order of `custom_styles.css` without affecting the order of anything else important (to minimize risk), this allows use to use one set of variables (colors, font stacks) for all our theme stylesheets, as discussed previously, is more performant since it only requires one request instead of 4, keeps everything neat and seperate in the source repo.

@dalthviz , what do you think?

Reply:
I will revert the changes I did and update the `.form-control` style over the theme. About unifying the styles and rethinking the order we load them is something we need to do but after thinking about it maybe is out of the scope of this PR. Could you open an issue to refactor the style order and in general style handling, please @CAM-Gerlach ?

Reply:
I still don't understand why we `custom_styles.css` and `custom_scripts.js`. Why can't we unify everything in the Sphinx theme repo?

Reply:
`custom_styles.css` manages styles that affect specific elements of the content that is being added in Spyder docs like the color of the icons/symbols the videos added with the directive and the interactive tutorial I think. `custom_scripts.js` was created to handle the tour load (and there I added too the handling of the version selector/dropdown).

Reply:
Ok, that makes a lot of sense. However, after looking at `custom_styles.css` I think some style declarations should be moved to the Sphinx repo instead, such as `iframe`, `#versions-header`, `#select-versions` and `#select-versions, #select-versions option`.

What do you think?

Reply:
Maybe `iframe` could be moved but the elements referring to versions are over this side (the template that renders the versions selector/dropdown is here)

Reply:
Should I move the `iframe` rules to the theme @ccordoba12 ?

Reply:
> maybe is out of the scope of this PR. Could you open an issue to refactor the style order and in general style handling, please @CAM-Gerlach ?

I agree, opened spyder-ide/spyder-docs-sphinx-theme#34

> Why can't we unify everything in the Sphinx theme repo?

For the same reason the theme is separate from the spyder-docs repo in the first place, because the purpose of a theme is to be modular and consumable by multiple projects (e.g. our future API docs, our other projects' docs, external projects, etc.) and to keep the core theming separate from the bespoke styles specific to the particular content we have at any given time.

> I think some style declarations should be moved to the Sphinx repo instead, such as iframe, #versions-header, #select-versions and #select-versions, #select-versions option.

At least to my recollection, the `iframe` rules are fairly specific to our use case and to our custom directive and container divs that implements them, that are hosted here in the theme.

As for the last three, it doesn't make sense to move the CSS styles without moving the version selector script and the HTML template, since they explicitly depend on them to do anything useful. I originally suggested this to @dalthviz on the basis that the version selector could be a useful common resource for other theme consumers, like the API docs, rather than something bespoke to our use case, but @dalthviz brought up that it would be "dead code" without the `sphinx-multiversion` dependency that not all theme users might require, and that if and when we need it elsewhere in the future, we could move it to the theme later.

Personally, I don't have a problem with support for optional dependencies in the theme, but since things are working as they are now and this is the final step to make the release, I'm not sure its worth blocking over a debate that doesn't affect the user-visible output and that we can easily resolve later.

Reply:
Ok, fair enough. Let's leave the css rules I mentioned here for the time being.

It's not a big issue and certainly not a blocker before the release.